examples/vision/ipynb/randaugment.ipynb

```diff
@@ -0,0 +1,590 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 0,
+   "metadata": {
+    "colab_type": "code"
+   },
+   "outputs": [],
+   "source": [
+    "simple_aug = tf.keras.Sequential(\n",
+    "    [\n",
+    "        layers.experimental.preprocessing.Resizing(IMAGE_SIZE, IMAGE_SIZE),\n",
+    "        layers.experimental.preprocessing.RandomFlip(\"horizontal\"),\n",
+    "        layers.experimental.preprocessing.RandomRotation(factor=0.02),\n",
+    "        layers.experimental.preprocessing.RandomZoom(\n",
+    "            height_factor=0.2, width_factor=0.2\n",
+    "        ),\n",
+    "    ]\n",
+    ")\n",
+    "\n",
+    "# Now, map the augmentation pipeline to our training dataset\n",
+    "train_ds_simple = (\n",
+    "    tf.data.Dataset.from_tensor_slices((x_train, y_train))\n",
+    "    .shuffle(BATCH_SIZE * 100)\n",
+    "    .batch(BATCH_SIZE)\n",
+    "    .map(lambda x, y: (simple_aug(x), y), num_parallel_calls=AUTO)\n",
+    "    .prefetch(AUTO)\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "colab_type": "text"
+   },
+   "source": [
+    "## Visualize the dataset augmented with RandAugment"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 0,
+   "metadata": {
+    "colab_type": "code"
+   },
+   "outputs": [],
+   "source": [
+    "sample_images, _ = next(iter(train_ds_rand))\n",
+    "plt.figure(figsize=(10, 10))\n",
+    "for i, image in enumerate(sample_images[:9]):\n",
+    "    ax = plt.subplot(3, 3, i + 1)\n",
+    "    plt.imshow(image.numpy().astype(\"int\"))\n",
+    "    plt.axis(\"off\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "colab_type": "text"
+   },
+   "source": [
+    "You are encouraged to run the above code block a couple of times to see different\n",
+    "variations."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "colab_type": "text"
+   },
+   "source": [
+    "## Visualize the dataset augmented with `simple_aug`"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 0,
+   "metadata": {
+    "colab_type": "code"
+   },
+   "outputs": [],
+   "source": [
+    "sample_images, _ = next(iter(train_ds_simple))\n",
+    "plt.figure(figsize=(10, 10))\n",
+    "for i, image in enumerate(sample_images[:9]):\n",
+    "    ax = plt.subplot(3, 3, i + 1)\n",
+    "    plt.imshow(image.numpy().astype(\"int\"))\n",
+    "    plt.axis(\"off\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "colab_type": "text"
+   },
+   "source": [
+    "## Define a model building utility function\n",
+    "\n",
+    "Now, we define a CNN model that is based on the\n",
+    "[ResNet50V2 architecture](https://arxiv.org/abs/1603.05027). Also,\n",
+    "notice that the network already has a rescaling layer inside it. This eliminates the need\n",
+    "to do any separate preprocessing on our dataset and is specifically very useful for\n",
```

Comment:
Since you are referring to ResNet50V2, the next part 
> notice that the network already has a rescaling layer inside it. This eliminates the need

It sounds almost like you are referring to Resnet50V2, that it has pre-processing inside, that is not the case, right?

Might be better to say something along the lines: 
> notice that a final network includes a rescaling layer inside it. This way, it eliminates the need to do any separate preprocessing

cc @fchollet @sayakpaul

Reply:
Right on. Thanks for spotting.